### ISSUE TITLE
Error install with Dokos v3 Erpnext v14

### ISSUE THREAD

**Issue by afrocubanito** [libracore/erpnextswiss#122]
### Expected behvauour
(fill in)

### Actual behaviour
(fill in)

### Observed in (version)
(master)

### Steps to reproduce
(fill in)

-----------------------
$ bench install-app erpnextswiss       

Installing erpnextswiss...
Updating DocTypes for erpnextswiss  : [=====                                   ] 13%An error occurred while installing erpnextswiss: Module import failed for EDI File, the DocType you're trying to open might be deleted.<br> Error: cannot import name 'send' from 'frappe.email.queue' (/home/frappe/frappe-bench/apps/frappe/frappe/email/queue.py)
Traceback (most recent call last):
  File "apps/frappe/frappe/modules/utils.py", line 289, in load_doctype_module
    doctype_python_modules[key] = frappe.get_module(module_name)
  File "apps/frappe/frappe/__init__.py", line 1351, in get_module
    return importlib.import_module(modulename)
  File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "apps/erpnextswiss/erpnextswiss/erpnextswiss/doctype/edi_file/edi_file.py", line 12, in <module>
    from frappe.email.queue import send
ImportError: cannot import name 'send' from 'frappe.email.queue' (apps/frappe/frappe/email/queue.py)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "apps/frappe/frappe/commands/site.py", line 433, in install_app
    _install_app(app, verbose=context.verbose, force=force)
  File "apps/frappe/frappe/installer.py", line 287, in install_app
    sync_for(name, force=force, reset_permissions=True)
  File "apps/frappe/frappe/model/sync.py", line 87, in sync_for
    import_file_by_path(
  File "apps/frappe/frappe/modules/import_file.py", line 149, in import_file_by_path
    import_doc(
  File "apps/frappe/frappe/modules/import_file.py", line 246, in import_doc
    doc.insert()
  File "apps/frappe/frappe/model/document.py", line 297, in insert
    self.run_post_save_methods()
  File "apps/frappe/frappe/model/document.py", line 1150, in run_post_save_methods
    self.run_method("on_update")
  File "apps/frappe/frappe/model/document.py", line 979, in run_method
    out = Document.hook(fn)(self, *args, **kwargs)
  File "apps/frappe/frappe/model/document.py", line 1348, in composer
    return composed(self, method, *args, **kwargs)
  File "apps/frappe/frappe/model/document.py", line 1330, in runner
    add_to_return_value(self, fn(self, *args, **kwargs))
  File "apps/frappe/frappe/model/document.py", line 976, in fn
    return method_object(*args, **kwargs)
  File "apps/frappe/frappe/core/doctype/doctype/doctype.py", line 417, in on_update
    self.run_module_method("on_doctype_update")
  File "apps/frappe/frappe/core/doctype/doctype/doctype.py", line 502, in run_module_method
    module = load_doctype_module(self.name, self.module)
  File "apps/frappe/frappe/modules/utils.py", line 293, in load_doctype_module
    raise ImportError(msg) from e
ImportError: Module import failed for EDI File, the DocType you're trying to open might be deleted.<br> Error: cannot import name 'send' from 'frappe.email.queue' (apps/frappe/frappe/email/queue.py)
---------

Thank you very much

**Comment by konradbr** [comment]
I can confirm having the same error while attempting to install on ERPnext v. 14:

```
erpnext 14.44.0
erpnextswiss 1.23.1
frappe 14.52.1
hrms 14.13.1
payments 0.0.1
```

**Comment by lasalesi** [comment]
Resolved by @latoile in PR #126
